### DISCUSSION TITLE
Override cmp key bindings

### DISCUSSION THREAD

**Discussion by zeeklop** [VonHeikemen/lsp-zero.nvim#22] [Q&A]
Hi there,

Fantastic job on this plugin. Made my config so much easier. I just have one problem, I like all the default key mappings, but I use
 `<C-k>` to move to a top split. How can I override only this specific mapping with my own? Is there a way other than overwriting `on_attach` function?

**Comment by VonHeikemen** [comment]
Using the `on_attach` function is the only way.

But I just pushed an update, `<Ctrl-k>` will not override an existing mapping set by the user.
